plugins/wasi_nn/wasinn_openvino.cpp

```diff
@@ -101,23 +101,11 @@ Expect<WASINN::ErrNo> setInput(WASINN::WasiNNEnvironment &Env,
 
   try {
     ov::element::Type InputType = ov::element::f32;
-    ov::Shape InputShape = {1, 224, 224, 3};
+    ov::Shape InputShape = {1, 3, 224, 224};
     ov::Tensor InputTensor =
         ov::Tensor(InputType, InputShape, Tensor.Tensor.data());
-    const ov::Layout InputLayout{"NHWC"};
-    ov::preprocess::PrePostProcessor PPP(GraphRef.OpenVINOModel);
-    PPP.input()
-        .tensor()
-        .set_shape(InputShape)
-        .set_element_type(InputType)
-        .set_layout(InputLayout);
-    PPP.input().preprocess().resize(
-        ov::preprocess::ResizeAlgorithm::RESIZE_LINEAR);
-    PPP.input().model().set_layout("NCHW");
-    PPP.output().tensor().set_element_type(ov::element::f32);
```

Comment:
this postprocessor removal would allows wasmedge to run models with non-fp32 outputs.
eg. https://github.com/openvinotoolkit/open_model_zoo/tree/cd7ebe313b69372763e76b82e5d24935308fece4/models/public/license-plate-recognition-barrier-0007